draft-vvv-webtransport-overview.md

```diff
@@ -189,6 +197,11 @@ sessions to the network endpoints that are not WebTransport servers.
 Any transport protocol used MUST provide a way for the server to filter the
 clients that can access it by the origin {{!RFC6454}}.
 
+Any transprot protocol used MUST provide a way for a server endpoint location to
```

Comment:
transprot => transport

Reply:
Fixed here and in another place I made the same typo.